apps/publications/src/app/pages/create-publication-page/create-publication-page.component.html

```diff
@@ -1 +1,38 @@
-<p>create-publication-page works!</p>
+<div class="user-theme">
+  <h1 class="page-subtitle d-flex">
+    <mat-icon class="add-icon">add_box</mat-icon>
+    <span class="ml-3">{{'CREATE_PUBLICATION.TITLE' | translate}}</span>
```

Comment:
![image](https://user-images.githubusercontent.com/32871167/113884174-10b86600-97bf-11eb-9f09-3989e5c426ac.png)
The page title is a little bit too high than we would like it, It should be on the same "line" as the icon, or at least centered

Reply:
Title and icon were aligned.